deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,61 @@
+{
+  'variables': {
+    'arm_fpu%': '',
+    'target_arch%': '',
+  },
+
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
```

Comment:
`direct_dependent_settings` defines settings that will be applied to other targets that directly depend on this target. So `<(node_lib_target_name)` will include this root directory. We can do `#include <simdutf.h>` in node lib sources.
```suggestion
      'include_dirs': ['.'],
      'direct_dependent_settings': {
        'include_dirs': [ '.' ],
      },
```

Reply:
Thanks! Updated the pull request.